include/ghostty.h

```diff
@@ -580,6 +580,7 @@ typedef enum {
   GHOSTTY_ACTION_TOGGLE_SPLIT_ZOOM,
   GHOSTTY_ACTION_PRESENT_TERMINAL,
   GHOSTTY_ACTION_SIZE_LIMIT,
+  GHOSTTY_ACTION_RESET_WINDOW_SIZE,
```

Comment:
Note that our keybind and action do not match the macOS menu item because macOS has very specific terminology it uses "Return to Default Size" within a very specific menu item (Window) that I don't think translates well verbatim to a general cross platform term. 

So, I use the macOS terminology/location for macOS, but choose `reset_window_size` as a more descriptive term for the core actions.